src/NServiceBus.Msmq.AcceptanceTests/EndpointCustomizationConfigurationExtensions.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace NServiceBus.Transport.Msmq.AcceptanceTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+    using AcceptanceTesting.Support;
+    using Hosting.Helpers;
+    using ObjectBuilder;
+
+    public static class EndpointCustomizationConfigurationExtensions
```

Comment:
Does the other downstreams copy paste this as well or is this only needed by MSMQ?